testing/integration_test.go

```diff
@@ -76,7 +76,7 @@ export NVM_DIR="$HOME/.nvm" >/dev/null
 
 # nvm install writes to stderr and stdout on successful install, so both are
```

Comment:
Update the comment?

Reply:
Updated the comment, and based on the comment, decided to keep the redirect of stdout.

Reply:
The comment still says "both are re-directed"? Didn't push the code?

Reply:
Yep -- forgot to push. Done.